aviatrix-controller-initialize/security_group.tf

```diff
@@ -0,0 +1,28 @@
+resource "aws_security_group" "AviatrixSecurityGroup2" {
+  name        = "${local.name_prefix}AviatrixSecurityGroup2"
+  description = "Aviatrix - Controller Security Group 2"
+  vpc_id      = var.vpc
+
+  tags = {
+    Name      = "${local.name_prefix}AviatrixSecurityGroup2"
+    Createdby = "Terraform+Aviatrix2"
+  }
+}
+
+resource "aws_security_group_rule" "ingress_rule" {
```

Comment:
Is this ingress rule required?

Reply:
This ingress rule is not required. Will remove it.